src/CoreWCF.RabbitMQ.Client/src/CoreWCF.RabbitMQ.Client.csproj

```diff
@@ -0,0 +1,20 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <AssemblyName>CoreWCF.RabbitMQ.Client</AssemblyName>
+    <PackageId>CoreWCF.RabbitMQ.Client</PackageId>
+    <RootNamespace />
+    <AllowUnsafeBlocks>True</AllowUnsafeBlocks>
+  </PropertyGroup>
+
+  <ItemGroup>
```

Comment:
@jonlouie I think we can avoid taking a dependency on the server side part and the common CoreWCF code.
Please have a look there to exclude common code.
https://github.com/CoreWCF/CoreWCF/pull/943/files#diff-9da24614831c308827a1ae533ffea392c97638c261dd42bd0f5226baa136d16e

Reply:
The client takes a dependency on the server side project because they share the RabbitMqConnectionSettings class (both server and client use it to handle endpoint URIs and create connections to a RabbitMQ host).

I also wanted to keep client separate from server, but besides duplicating the class, no other immediate solutions came to mind.

Reply:
I see. But this dependency leads to:
CoreWCF.RabbitMQ.Client => CoreWCF.RabbitMQ => CoreWCF.Primitives => ASP.NET Core 2.1
You could either duplicate code (this is the solution i've chosen for the Kafka implementation) or move the common RabbitMQ code to the client package and make the server take a dependency on the client. this would avoid clients to pull ASP.NET Core.